src/web/Button.tsx

```diff
@@ -234,7 +234,7 @@ export class Button extends ButtonBase {
 
         // Touch has left the button, cancel the longpress handler.
         if (this._isMouseOver && this._longPressTimer) {
-            Timers.clearTimeout(this._longPressTimer);
```

Comment:
Looks like a merge conflict occurred here. I recently updated all of the reactxp code to use Timers.clearTimeout consistently.

Reply:
✅